src/components/Logo/index.ts

```diff
@@ -0,0 +1 @@
+export { default as Logo } from "./Logo"
```

Comment:
I think it was my mistake here, because we actually set the name of the default export to `Logo`. Still, I've mentioned that this could be better (although I admit, it's actually not) in case we would want to export `LogoProps`, but we don't need to export it right now.

I would propose to not export types if it's not necessary, and we don't use `HeaderProps` and `LogoProps` outside of the component. What do you think about that?

If you agree then we should remove the changes from b9ed5451411f44421613e1da73b6201d840a6b5d.

Reply:
Reverted

Ref commit: 3b161e4e

As a side note: exporting props type definition might be useful. In my opinion better approach would be to always export it instead of exporting on demand - just for consistency. Nevertheless the rules and guidelines of structuring the project are not formally standardized in any way so currently it's a matter of preference. I don't stickle against reverting changes at the moment. If we were to define such standards I would opt for my point of view, just for consistency as I said.